src/app/system/manage-dashboards/manage-dashboards.component.html

```
@@ -0,0 +1,434 @@
+              <fa-icon icon="save" class="m-r-5"></fa-icon>
+              {{ 'labels.buttons.Save Changes' | translate }}
+            </button>
+          </div>
+
+          <!-- Widgets Config Table -->
+          <mat-card class="widgets-table-card mat-elevation-z1 p-0">
+            <div class="table-responsive">
+              <table class="mifos-table">
+                <thead>
+                  <tr>
+                    <th style="width: 70px">{{ 'labels.heading.Status' | translate }}</th>
```

HTMLHint (1.9.2)

[warning] Inline style [  style="width: 70px" ] cannot be used.

(inline-style-disabled)